src/mono/mono/metadata/filewatcher.h

```diff
@@ -20,16 +20,6 @@
 #include <unistd.h>
 #endif
 
-#if !ENABLE_NETCORE
-
-ICALL_EXPORT
-gint ves_icall_System_IO_FSW_SupportsFSW (void);
-
-ICALL_EXPORT
-int ves_icall_System_IO_KqueueMonitor_kevent_notimeout (int *kq, gpointer changelist, int nchanges, gpointer eventlist, int nevents);
-
-#endif
-
 #ifdef HOST_IOS // This will obsoleted by System.Native as soon as it's ported to iOS
 MONO_API char* SystemNative_RealPath(const char* path);
 MONO_API void SystemNative_Sync (void);
```

Comment:
this shouldn't apply anymore here either so we can get rid of filewatcher.h completely.

Reply:
Will do.